contracts/src/core/BatcherPaymentService.sol

```diff
@@ -21,6 +21,10 @@ contract BatcherPaymentService is
     // EVENTS
     event PaymentReceived(address indexed sender, uint256 amount);
     event FundsWithdrawn(address indexed recipient, uint256 amount);
+    event TaskCreated(bytes32 indexed batchMerkleRoot, string batchDataPointer);
+    event BalanceLocked(address indexed user);
+    event BalanceUnlocked(address indexed user, uint256 unlockBlock);
```

Comment:
should emit in `receive()` and `withdraw()` when that PR is merged

Reply:
Solved in [e9e442d](https://github.com/yetanotherco/aligned_layer/pull/840/commits/e9e442d6699faf0b643ecfd516bbf57aa2dbe4c0). Receive doesn't lock.